src/main.py

```diff
@@ -104,13 +98,27 @@ def render_page(sensors):
     measure = 'GET /?measure=on' in request
 
     # note that LED is on on low signal
+    measurements = []
```

Comment:
Measurements can happen "concurrently" as the sensors are all independent of each other. We call each `measure` method (which is now async.) and append the `Future` it returns to a list. This does not block or wait for `measure` to complete.